sharding/indexHashedNodesCoordinator_test.go

```diff
@@ -500,6 +502,125 @@ func TestIndexHashedGroupSelector_ComputeValidatorsGroupTest6From10ValidatorsSho
 	assert.Equal(t, validator4, list2[5])
 }
 
+func TestIndexHashedGroupSelector_ComputeValidatorsGroup400of400For10locksNoMemoization(t *testing.T) {
```

Comment:
Memoization?

Reply:
NoMemoization because the get always returns false. The name memoization should be appropiate: https://en.wikipedia.org/wiki/Memoization